sql/core/src/test/resources/sql-tests/inputs/with.sql

```diff
@@ -0,0 +1,14 @@
+create temporary view t as select * from values 0, 1, 2 as t(id);
```

Comment:
Here, t1 and t2 is base tables.
The followings are used CTEs and to check if base table or previous CTE is used correctly.